appengine/appidentity/README.md

```diff
@@ -0,0 +1,12 @@
+# App Identity sample for Google App Engine
+This sample demonstrates how to use the App Identity APIs on Google App Engine
+
+## Setup
+1. Update the <application> tag in src/main/webapp/WEB-INF/appengine-web.xml with your project name
+1. Update the <version> tag in src/main/webapp/WEB-INF/appengine-web.xml with your version name
```

Comment:
I believe the appengine maven plugin throws an error if you don't have both application and version tags.